.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+
+    services:
+      toxiproxy:
+        image: shopify/toxiproxy
+        ports:
+          - 8474:8474
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - { ruby: 2.5.8 }
+        - { ruby: 2.6.6 }
+        - { ruby: 2.7.2 }
+        - { ruby: head, allow-failure: true }
```

Comment:
it fails on `head`, it has nothing to do with Actions though. One needs to add `rexml` explicitly to the `gemspec` for Ruby 3.0

Reply:
(It's no loger a "standard gem", but rather a "bundled gem")

Reply:
Opened #16 for that.